locale/fr/manageiq.po

```diff
@@ -31747,13 +31749,13 @@ msgstr[1] ""
 #: ../app/controllers/application_controller/sysprep_answer_file.rb:16
 msgid "Error during Sysprep \"%{params}\" file upload: %{message}"
 msgstr ""
-"Erreur lors du téléchargement du fichier \"%{params}\" de Sysprep : %{message}"
+"Erreur lors du transfert du fichier \"%{params}\" de Sysprep : %{message}"
 
 #: ../app/controllers/application_controller/user_script_file.rb:17
 msgid "Error during User Script \"%{params}\" file upload: %{message}"
 msgstr ""
-"Erreur lors du téléchargement de fichier \"%{params}\" de script utilisateur :"
-" %{message}"
+"Erreur lors du transfert du fichier \"%{params}\" de script utilisateur : %{"
+"message}"
```

Comment:
Look like there are still fixes needed though :(

Reply:
and missing tests I guess 😢

Reply:
hmm, were these already broken?

```
 #: ../app/controllers/vm_common.rb:1324
 msgid "%{action} \"%{item_name}\" for %{vm_or_template} \"%{name}\""
@@ -576,10 +576,10 @@ msgid "%{action} initiated for %{count} %{model} from the %{product} Database"
 msgid_plural ""
 "%{action} initiated for %{count} %{models} from the %{product} Database"
 msgstr[0] ""
-"%{action} lancée pour %{count} %{model} à partir de la base de données %{"
+"%{action} initiée pour %{count} %{model} à partir de la base de données %{"
 "product}"
 msgstr[1] ""
-"%{action} lancée pour %{count} %{models} à partir de la base de données %{"
+"%{action} initiée pour %{count} %{models} à partir de la base de données %{"
 "product}"
```

Reply:
These might be ok, because the final string will be correct...I think the issue came when arbitrary stuff was added between (like "\n" or " ")

Reply:
It looks ok...the placeholders spec is checking this and seems to be ok.

from:
```diff
diff --git a/spec/shared/i18n/placeholders.rb b/spec/shared/i18n/placeholders.rb
index 3ba2054b90..94a2764db5 100644
--- a/spec/shared/i18n/placeholders.rb
+++ b/spec/shared/i18n/placeholders.rb
@@ -32,6 +32,9 @@ shared_examples :placeholders do |dir|
       plural_unsupported = (po.pluralisation_rule.call(3) == 0)

       po.data.each do |original, translation|
+        puts "before: #{original.inspect}"
+        puts "after:  #{translation.inspect}"
+
```

```
before: "Error during User Script \"%{params}\" file upload: %{message}"
after:  "Erreur lors du transfert du fichier \"%{params}\" de script utilisateur : %{message}"
```

```
before: "%{action} initiated for %{count} %{model} from the %{product} Database"
after:  "%{action} initiée pour %{count} %{model} à partir de la base de données %{product}"
```

```
before: "%{action} initiated for %{count} %{models} from the %{product} Database"
after:  "%{action} initiée pour %{count} %{models} à partir de la base de données %{product}"
```